front/components/Button/index.tsx

```diff
@@ -61,6 +61,7 @@ const Button = React.forwardRef(({
       disabled={disabled}
       variant={variant}
       disableElevation
+      data-testid="button"
```

Comment:
We should probably avoid this kind of `data-testid` in reusable components. `data-testid` should be the last resort